v2/passwordless/pre-built-ui/setup/core/self-hosted-with-aws.mdx

```diff
@@ -0,0 +1,20 @@
+---  
```

Comment:
Please modify the [sidebar.json](https://github.com/supertokens/docs/blob/a4e2261fb53b295834f55239747e4a40835d18c6/v2/passwordless/sidebars.js) to add this page